content/_data/changelogs/weekly.yml

```
@@ -8542,6 +8542,64 @@
 
   # pull: 4875 (PR title: Add new line at EOF in config.xml)
 
+- version: '2.251'
+  date: 2020-08-05
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4892
+    issue: 63276
+    authors:
+    - fqueiruga
+    message: |-
+      Empty installed plugins table text is readable again (regression in 2.249).
+  - type: major bug
+    category: regression
+    pull: 4870
+    issue: 63180
+    authors:
+    - fqueiruga
+    message: |-
+      Restore wrapping tabs into multiple lines instead of overflowing (regression in 2.248).
+  - type: bug
```

https://github.com/jenkinsci/jenkins/pull/4893
```suggestion
  - type: bug
    category: bug
    pull: 4893
    issue: 63232
    authors:
    - Wadeck
    message: |-
      Fix a regression in Build Time Trend Page
  - type: bug
```